examples/app_multi_uart_demo/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(app_multi_uart_demo)
+
+set(APP_HW_TARGET XK-EVK-XU316)
+set(APP_COMPILER_FLAGS -g -O3 -save-temps -DMUART_RX_CHAN_COUNT=8 -DMUART_TX_CHAN_COUNT=8)
```

Comment:
not sure we want -save-temps by default?

Reply:
just kept it as it was there, can remove

Reply:
I would I think it breaks if parallelised.